src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,38 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+
+interface Props {
+  selectedTodo: Todo | undefined;
+  onCloseModal: () => void,
```

Comment:
`selectedTodo: Todo | undefined;`
if you wait for object better use 'null' not 'udefined'